cypress/e2e/annual-review/worker.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Emails sent on key dates
+
+  As a provider,
+  I should be included in the annual review process if I have been published for over a month,
+  so that users can see my updated details
+
+  As a provider,
+  I should not be included in the annual review process if I have been published for under a month,
+  so that I do not have to review my details that have recently been reviewed
+
+  As consular staff,
+  I should only review providers that have been published for over a month,
+  So that I do not have to verify details that have been recently reviewed.
```

Comment:
Haven't seen this in the project before. Is this valid gherkin?

Reply:
Would it be better to give the summarise the test after the `Scenario:` keyword instead?

Reply:
yes, it is. https://cucumber.io/docs/gherkin/reference/#feature

In this case, not really, this acts as feature documentation and an overview of the scenarios. The scenarios themselves are readable without additional context.

see my comment here: https://github.com/UKForeignOffice/lists/pull/579/files#r1114762019